x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -977,4 +979,56 @@ export const FIELD: Record<string, FieldMeta> = {
       required: true,
     }),
   },
+  [ConfigKey.PLAYWRIGHT_OPTIONS]: {
+    fieldKey: ConfigKey.PLAYWRIGHT_OPTIONS,
+    component: JSONEditor,
+    label: i18n.translate('xpack.synthetics.monitorConfig.playwrightOptions.label', {
+      defaultMessage: 'Playwright options',
+    }),
+    helpText: (
+      <span>
+        {i18n.translate('xpack.synthetics.monitorConfig.playwrightOptions.helpText', {
+          defaultMessage: 'Configure Playwright agent with custom options. ',
+        })}
+        <EuiLink
+          href={getDocLinks()?.links?.observability?.syntheticsCommandReference}
```

Comment:
It's difficult to tell based on the screenshot, but would [Synthetic tests configuration](elastic.co/guide/en/observability/current/synthetics-configuration.html) be a better place to link to?

Reply:
@colleenmcginnis I think you're right. I was looking for this section in the documentation and couldn't find it.
One caveat though, the link to playwright documentation seems to be broken, it should probably point to https://playwright.dev/docs/test-configuration. Do you want me to create a PR in the docs to fix that?

Reply:
I can fix it! Thanks for letting me know.

Reply:
Thanks! Changed the link to point to right docs